docs/content/preview/reference/configuration/insecure-yugabyted-examples.md

```diff
@@ -0,0 +1,939 @@
+
+#### Syntax
+
+```sh
+Usage: yugabyted demo destroy [flags]
+```
+
+#### Flags
+
+-h | --help
+: Print the help message and exit.
+
+--config *config-file*
+: The path to the configuration file of the yugabyted server whose logs are desired.
+
+--data_dir *data-directory*
+: The data directory for the yugabyted server whose logs are desired.
+
+--base_dir *base-directory*
+: The base directory for the yugabyted server whose logs are desired.
+
+--log_dir *log-directory*
+: The directory to store yugabyted logs. Must be an absolute path. This flag controls where the logs of the YugabyteDB nodes are stored.
+
+-----
+
+## Environment Variables
+
+In the case of multi-node deployments, all nodes should have similar environment variables.
+
+Changing the values of the environment variables after the first run has no effect.
+
+### YSQL
+
+Set `YSQL_PASSWORD` to use the cluster in enforced authentication mode.
+
+Combinations of environment variables and their uses.
+
+- `YSQL_PASSWORD`
+
+  Update the default yugabyte user's password.
+
+- `YSQL_PASSWORD, YSQL_DB`
+
+  Update the default yugabyte user's password and create `YSQL_DB` named DB.
+
+- `YSQL_PASSWORD, YSQL_USER`
+
+  Create `YSQL_USER` named user and DB with password `YSQL_PASSWORD`.
+
+- `YSQL_USER`
+
+  Create `YSQL_USER` named user and DB with password `YSQL_USER`.
+
+- `YSQL_USER, YSQL_DB`
+
+  Create `YSQL_USER` named user with password `YSQL_USER` and `YSQL_DB` named DB.
+
+- `YSQL_DB`
+
+  Create `YSQL_DB` named DB.
+
+- `YSQL_USER, YSQL_PASSWORD, YSQL_DB`
+
+  Create `YSQL_USER` named user with password `YSQL_PASSWORD` and `YSQL_DB` named DB.
+
+### YCQL
+
+Set `YCQL_USER` or `YCQL_PASSWORD` to use the cluster in enforced authentication mode.
+
+Combinations of environment variables and their uses.
+
+- `YCQL_PASSWORD`
+
+  Update the default cassandra user's password.
+
+- `YCQL_PASSWORD, YCQL_KEYSPACE`
+
+  Update the default cassandra user's password and create `YCQL_KEYSPACE` named keyspace.
+
+- `YCQL_PASSWORD, YCQL_USER`
+
+  Create `YCQL_USER` named user and DB with password `YCQL_PASSWORD`.
+
+- `YCQL_USER`
+
+  Create `YCQL_USER` named user and DB with password `YCQL_USER`.
+
+- `YCQL_USER, YCQL_KEYSPACE`
+
+  Create `YCQL_USER` named user with password `YCQL_USER` and `YCQL_USER` named keyspace.
+
+- `YCQL_KEYSPACE`
+
+  Create `YCQL_KEYSPACE` named keyspace.
+
+- `YCQL_USER, YCQL_PASSWORD, YCQL_KEYSPACE`
+
+  Create `YCQL_USER` named user with password `YCQL_PASSWORD` and `YCQL_KEYSPACE` named keyspace.
+
+-----
+
+## Examples
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../yugabyted/#examples" class="nav-link">
+      <!-- <i class="fa-brands fa-apple" aria-hidden="true"></i> -->
+    Secure Cluster
+    </a>
+  </li>
+  <li class="active">
+    <a href="../insecure-yugabyted-examples/#examples" class="nav-link">
+      <!-- <i class="fa-brands fa-linux" aria-hidden="true"></i> -->
+    Insecure Cluster
```

Comment:
Unsecured cluster

Reply:
Keeping the `secure` and `insecure` terminology.